sci-electronics/spice/spice-3.5.5-r4.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
```

Comment:
For bumping the EAPI we should move `virtual/pkgconfig` to `BDEPEND` (which is a new thing introduced in EAPI 7 for build time dependencies that are __executed__ during the build).

Reply:
Done